SAMtags.tex

```diff
@@ -312,9 +312,6 @@ \subsection{Barcodes}
 The qualities here may have been corrected (Raw bases and qualities can be stored in {\tt OX} and {\tt BZ} respectively.)
 The lengths of the {\tt QX} and the {\tt RX} tags must match.
 In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the quality strings with a space (`{\tt \textvisiblespace}') between the different strings.
-
-\item[RT:Z:\tagvalue{sequence}]
-Deprecated alternative to {\tt BC} tag originally used at Sanger.
```

Comment:
I'd keep a sentence here explaining that while we no longer condone the use of RT for any purpose, it cannot be used for any new purpose due to the fact that it had been used in the past.